internal/painter/svg_cache.go

```diff
@@ -32,7 +32,13 @@ func init() {
 
 // GetAspect looks up an aspect ratio of an image
 func GetAspect(img *canvas.Image) float32 {
-	aspect := aspects[img.Resource]
+	aspect := float32(0.0)
```

Comment:
A named return variable would avoid this line ;)

Reply:
True - but something about comparing the value of a not-visibly-initialised variable 8 lines lower made me feel uneasy. Possibly the C developer in me?

Reply:
Both points are very valid, yet I think a named return value might look cleaner. Do as you please :)